src/middlewared/middlewared/alert/source/collectd.py

```diff
@@ -49,7 +49,10 @@ def check_sync(self):
         alerts = []
         for k, v in list(data.items()):
             if k == "ctl-ha/disk_octets":
-                text = "CTL HA link is actively used, check initiators connectivity."
+                text = (
+                    "CTL HA link is actively used. Please check that all your iSCSI and FC initiators support ALUA and "
```

Comment:
"actively used" is kind of meaningless.  "in use" might be what you mean, but I wrote that before and it didn't change, so maybe it is something else.
For the second part, avoid "your", it can just be removed.  With those changes, this becomes a lot more helpful to the user.